### PR TITLE
Disable Solidity EVM

### PR DESCRIPTION
**Pull request by hanssv**
It was of great use while developing, but it is not well tested enough
to be part of consensus, thus it is disabled.

See also https://github.com/aeternity/protocol/pull/271

https://www.pivotaltracker.com/story/show/162082161

### CODE REVIEW

File: apps/aecontract/src/aect_channel_contract.erl [hunk 1]
```diff
@@ -33,6 +33,11 @@ run_new(ContractPubKey, Call, CallData, Trees0, OnChainTrees,
     CallStack = [], %% TODO: should we have a call stack for create_tx also
                     %% when creating a contract in a contract.
     VmVersion = aect_contracts:vm_version(Contract),
+    %% Assert VmVersion before running!
+    case VmVersion of
+        ?AEVM_01_Sophia_01 -> ok
```
Comment (lucafavatella) [inline apps/aecontract/src/aect_channel_contract.erl]: What is under consensus seems to me to be `:run`, and the corresponding check tx. /me checking.

Comment (lucafavatella) [reply apps/aecontract/src/aect_channel_contract.erl]: I am drafting VM check for state channel force progress in [temporary branch](https://github.com/aeternity/epoch/compare/PT-162082161-disable_solidity_evm...PT-162082161-disable_solidity_evm--sc-fp).

Comment (lucafavatella) [reply apps/aecontract/src/aect_channel_contract.erl]: https://github.com/aeternity/epoch/pull/1807

Comment (hanssv) [reply apps/aecontract/src/aect_channel_contract.erl]: Well spotted! I was slightly fooled by Dimitar! I've merged Luca's branch.

**Pull request merged**